app/controllers/admin/admin_controller.rb

```diff
@@ -1,10 +1,25 @@
-class Admin::AdminController < SlicesController
+class Admin::AdminController < ActionController::Base
   before_filter :authenticate_admin!
+  around_filter :set_locale
+
+  append_view_path(File.join(Rails.root, *%w[app slices]))
 
   protected
 
   def presenter_class(page_class)
     Object.const_get("#{page_class.name}Presenter")
   end
+
+  private
+
+  def request_locale
+    params[:locale]
+  end
+
+  def set_locale
+    I18n.with_locale(request_locale || I18n.default_locale) do
+      yield
+    end
```

Comment:
Yeah it can, just a style thing. Not sure if rubocop has a rule about it.

Actually this change shouldn't be in this PR anyway, it should be in #107 - the commit needs splitting. I'll move it across. Thanks for commenting so I noticed!